virttest/utils_misc.py

```diff
@@ -312,6 +312,28 @@ def find_substring(string, pattern1, pattern2=None):
     return ret[0]
 
 
+def get_match_count(check_path, key_message):
+    """
+    Get expected messages count in path
+
+    :param check_path: file path to be checked
+    :param key_message: key message that needs to be captured
+    :return count: the count of key message
+    """
+    count = 0
+    try:
+        with open(check_path, 'r', encoding='ISO-8859-1') as fp:
```

Comment:
Add encoding to the function parameter for extensible usage.

Reply:
THanks done